### ISSUE TITLE
Even Better Way to Bypass Geolocation (no networking required)

### ISSUE THREAD

**Issue by SentToDevNull** [true-systems/om5p-ac-v2-unlocker#3]
Amazing how stupidly simple this was. I was about to set up an OpenVPN service on another older router in order to trick the geolocation feature of the om5p-ac-v2 Cloudtrax router, and rebooting that older router, my network was accidentally switched to that provided by the Cloudtrax router. I refreshed the LuCi page started by my other router, but found that it was the Cloudtrax router and had no password set up. Just typing a username of "root" and a password of "root" on the "http://192.168.1.1/cgi-bin/luci" page of the Cloudtrax router (oops; they forgot to disable that) got me in, and once I changed the password, I could SSH into it. No VPN needed. Then I could enter the commands for unlocking it.

TLDR: Hook it up directly to your computer and go to "http://192.168.1.1/cgi-bin/luci". Change the admin password under the "Administration" page. SSH into it. Then, just follow the instructions on "https://github.com/true-systems/om5p-ac-v2-unlocker/wiki/Unlocking-using-SSH-connection". Voila!

**Comment by SentToDevNull** [comment]
May have posted this too soon. I played around with it for a few hours first. I'm going to try to replicate it.

**Comment by SentToDevNull** [comment]
Reflashed the router with stock firmware to try to replicate. Apparently it worked the first time due to something I did earlier. I'll edit the original post if I can figure out what that was.

**Comment by pepe2k** [comment]
> Reflashed the router with stock firmware to try to replicate. Apparently it worked the first time due to something I did earlier. I'll edit the original post if I can figure out what that was.

Have you flashed it with OpenWrt or LEDE before maybe?

Cheers,
Piotr
